features/step_definitions/login_steps.rb

```diff
@@ -0,0 +1,5 @@
+Given("following user exist") do |table|
+  table.hashes.each do |user|
+    FactoryBot.create(:user, user)
```

Comment:
Make use of short command for Factorybot ```  create(:user, user) ```

Reply:
This step definition already exists in register_steps!